src/pages/revision/[id].tsx

```diff
@@ -153,19 +190,53 @@ const Revision = () => {
               >
                 <WikiActionBar wiki={wiki?.content[0]} />
                 {wiki ? (
-                  <Box>
+                  <Box w="100%">
                     <Flex
-                      w="100%"
-                      justify="space-between"
-                      direction={{ base: 'column', md: 'row' }}
+                      display={{ lg: 'flex', base: 'none', md: 'flex' }}
+                      flexDirection={{
+                        base: 'column-reverse',
+                        lg: 'row',
+                        md: 'row',
+                      }}
                     >
-                      <WikiMainContent wiki={wiki.content[0]} />
+                      <WikiMainContent
+                        wiki={wiki.content[0]}
+                        mobileView={false}
+                      />
                       <WikiInsights
                         dateTime={wiki.datetime}
                         wiki={wiki.content[0]}
                         ipfs={wiki.ipfs}
                       />
                     </Flex>
+                    <Flex
+                      display={{ lg: 'none', base: 'block', md: 'none' }}
+                      flexDirection={{
+                        base: 'column-reverse',
+                        lg: 'row',
+                        md: 'row',
```

Comment:
Same here, no need for `lg`
```suggestion
                        md: 'row',
```